src/boxGrid/boxGridBox.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  topLabel?: string;
+  label: string;
+  bottomText: string;
+  color: 'pink' | 'gray' | 'yellow' | 'blue';
```

Comment:
Just a though, but this could be optional and defaults to "primary" color.

Reply:
I suppose a default color wouldn't hurt. Probably not the primary color, though, since it's a pretty dark blue in our case, which would be pretty disconnected from this initial set of bright colors that has been like this since the earliest stages of the front page.